setup.cfg

```diff
@@ -0,0 +1,49 @@
+[tool:pytest]
+filterwarnings=
+    error
+
+[mypy]
```

Comment:
I'd be tempted to keep this separate in `.mypy.ini` as it's a substantial amount of config.